modules/claims_api/spec/sidekiq/v2/poa_form_builder_job_spec.rb

```diff
@@ -23,45 +26,27 @@
           consentLimits: %w[DRUG_ABUSE SICKLE_CELL],
           veteran: {
             address: {
-              numberAndStreet: '2719 Hyperion Ave',
+              addressLine1: '2719 Hyperion Ave',
               city: 'Los Angeles',
-              state: 'CA',
+              stateCode: 'CA',
               country: 'US',
-              zipFirstFive: '92264'
+              zipCode: '92264'
             },
             phone: {
               areaCode: '555',
               phoneNumber: '5551337'
             }
           },
-          claimant: {
```

Comment:
shouldn't have `claimant` attribute for Veteran claimant specific test. Non-Veteran claimant test has the `claimant` attribute and tests that functionality.